src/components/Loading/Page/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component, Fragment } from 'react';
```

Comment:
This file can be removed, duplicate with `Loading/index.jsx`

Reply:
ok